addons/website/tools.py

```diff
@@ -84,6 +84,12 @@ def __init__(self, env, **kw):
         app.get_db_router = app.bind = app.match = app
         if not kw.get('routing', True):
             app.match = werkzeugRaiseNotFound
+
+        lang = kw.get('lang')
+        if not lang:
+            lang_code = kw.get('context', {}).get('lang', env.context.get('lang', 'en_US'))
+            lang = env['res.lang'].search([('code', '=', lang_code)])
```

Comment:
use `_lang_get` to retrieve a language from its code (and also `en_US` above)

Reply:
Done, forgot that one as only used in test